### PR TITLE
fixing export locations / test successful importing

### PR DESCRIPTION
**Pull request by alex-wilmer**
I'm kind of happy how this experiment turned out? I wanted to change the way arranger/components were being exported and I knew it was going to break stuff. 

this PR in its current state doesn't cover that entirely but I think this could be used as a decent model for integration tests moving forward.

### CODE REVIEW

File: modules/components/src/Admin/Dashboard.js [hunk 1]
```diff
@@ -519,7 +524,7 @@ class Dashboard extends React.Component {
           />
           <Route
             exact
-            path="/projects/:projectId/:index"
+            path={basePath + '/:projectId/:index'}
```
Comment (Jephuff) [inline modules/components/src/Admin/Dashboard.js]: Can you avoid concating basePath and passing it around by making a router for the dashboard and setting `basename` on it?

Comment (alex-wilmer) [reply modules/components/src/Admin/Dashboard.js]: I didn't think you could nest routers. I'll try it

Comment (alex-wilmer) [reply modules/components/src/Admin/Dashboard.js]: this does seem to work. is this what you mean?

```
  render() {
    return (
      <BrowserRouter>
      <div>
        <Link to="/admin">go</Link>
        <Route path="/admin" render={({ match }) =>
          <BrowserRouter basename={match.url}>
            <div>
              <Route path="/" render={() => <div>sup</div>} />
            </div>
          </BrowserRouter>
        } />
      </div>
      </BrowserRouter>
    );
  }
```

Comment (alex-wilmer) [reply modules/components/src/Admin/Dashboard.js]: this seems far better than what the official docs suggest for nested routing

Comment (Jephuff) [reply modules/components/src/Admin/Dashboard.js]: yeah thats exactly what I was thinking :) it works? yay! 
whats the official method?

Comment (alex-wilmer) [reply modules/components/src/Admin/Dashboard.js]: they just don't go that far to say nest routers. they're probably being conservative considering you rely on nesting context 

https://reacttraining.com/react-router/core/guides/philosophy/nested-routes

**Pull request merged**